validators/register-validator-node.md

```diff
@@ -355,19 +373,20 @@ sudo docker-compose stop
 
 ## 3. Setup SKALE Node with SKALE Node CLI
 
-After Setting up SGX Wallet and create certifications, validators can download the SKALE Node CLI executables register and maintain your SKALE node. This process downloads docker container images from docker hub and spins up SKALE Node functionalities. Some of the base containers such as SKALE Admin, Bounty, SLA, TransactionManager will be created during installation for each node.
+After Setting up SGX Wallet and create certifications, validators can download the SKALE Node CLI executables register and maintain your SKALE node. This process downloads docker container images from docker hub and spins up SKALE Node functionalities. Some base containers such as SKALE Admin, Bounty, SLA, TransactionManager will be created during installation for each node.
 
-See the SKALE Node CLI code and documentation on [**GitHub**](https://github.com/skalenetwork/skale-node-cli)*‍
+See the SKALE Node CLI code and documentation on [**GitHub**](https://github.com/skalenetwork/skale-node-cli)\*‍
 
 This document contains instructions on how to get started with the SKALE Node CLI.
 
 ### **Prerequisites**
 
 -   A Linux x86_64 machine
 -   SGX-enabled Intel processor
--   Ports 22, 8080, 9100, and 10000-11000, and ICMP IPv4 open for all
+-   Ports 22, 8080, 9100, and 10000–11000, and ICMP IPv4 open for all
+-   SSL Certificates
 -   Ubuntu 18.04 or later LTS
--   2TB attached storage main-net (200gb devnet)
+-   2TB attached storage main-net (200gb Testnet)
```

Comment:
can we also add notification here (unmounted) We had issues with mounted disks during initialization

Reply:
added unmounted note